src/test/unit_tests/test_nc_health.js

```diff
@@ -635,6 +639,24 @@ mocha.describe('nsfs nc health', function() {
             await fs_utils.file_delete(Health.config_fs.system_json_path);
             assert_config_dir_status(health_status, valid_system_json.config_directory);
         });
+
+        mocha.it('invalid blocked hosts', async function() {
```

Comment:
I think I don't understand the title of this test... ("invalid blocked hosts")
We are removing the `system.json` file, right? So it tests inaccessible system.json?

Reply:
no, it means that we never started the config directory upgrade but the host is already upgraded, therefore the host is blocked for updates to the config directory.